src/components/forms/CheckboxFormField.js

```diff
@@ -6,7 +6,7 @@ import FormErrorMessage from "./FormErrorMessage";
 import { Colors, Typography } from "../../constants/styles";
 import FontForgeIcon from "../common/FontForgeIcon";
 
-const CheckboxFormField = ({ name, text }) => {
+const CheckboxFormField = ({ name, text, marginLeft }) => {
```

Comment:
Chyba lepiej już zrobić to bardziej ogólnie i przekazać cały styl, a nie tylko jeden margines